app/mailers/claim_submission_mailer.rb

```diff
@@ -8,7 +8,7 @@ def notify(claim)
       LAA_case_reference: case_reference,
       UFN: unique_file_number,
       main_defendant_name: defendant_name,
-      maat_id: maat_id,
+      defendant_id: defendant_id_string,
```

Comment:
Good to call this out and normalise it as both technical and non technical will be seeing this because of the templates. Have updated in both places now thank you